test/spec/bidmanager_spec.js

```diff
@@ -465,7 +465,9 @@ describe('bidmanager.js', function () {
     before(() => {
       $$PREBID_GLOBAL$$.adUnits = fixtures.getAdUnits();
     });
-
+    after(() =>{
+      $$PREBID_GLOBAL$$.adUnits = [];
```

Comment:
Hi @mkendall07 ,

Yeah, this line is definitely not required since the `before` in `adUnits_spec` takes care of resetting the value, but I just put it in for the sake of cleanliness; in my opinion, since this `describe` has a `before` that sets a value on the global object, it seems reasonable to reset the value in an `after`. However, it's not necessary.
Would you like me to take it out?